SDK_versioned_docs/version-3.0.0/guides/liquidity/06-removing-liquidity.md

```diff
@@ -0,0 +1,100 @@
+---
+id: removing
+title: Removing Liquidity
+---
+
+## Removing Liquidity from a Position
+
+To retrieve the remove liquidity calldata, use the function `removeCallParameters`, which takes in a `Position`, which we defined earlier, and `RemoveLiquidityOptions`. The reference for `RemoveLiquidityOptions` is:
```

Comment:
"use the function `removeCallParameters` with the `Position` parameter we defined earlier and `RemoveLiquidityOptions`"

Reply:
do these reference file exist in Technical Reference folder?  https://uniswap-docs-8ukoaa9fd-uniswap.vercel.app/sdk/reference/interfaces/RemoveLiquidityOptions

should we link to that page instead of copying code here?  Does the code change over time so this copy would become stale?

Reply:
I think its nice to have but I will also link out to it.